### ISSUE TITLE
Add Function: Mute output

### ISSUE THREAD

**Issue by Avnsx** [ultrafunkamsterdam/undetected-chromedriver#62]
Hey,
I've been running chromedriver for a while now and I really care alot about a clean output.
It got kinda annoying that chromedriver prints `Selenium patched. Safe to import Chrome / ChromeOptions` every time it's executed, could you add an option to mute that or just entirely remove it from the source, please 👀👍

**Comment by ultrafunkamsterdam** [comment]
Yeah sure, but it would raise a bunch bunch of "issues" when it's not clear whether patching took place or not. However I'm working on v2 which uses logging module and thus easily disabled by something like ```logging.getLogger('UC').disabled = True```
